docs/topics/migrations.txt

```diff
@@ -305,6 +305,27 @@ migrations are treated no differently from any other migration.
 
 .. _migration-history-consistency:
 
+Non-atomic migrations
+---------------------
+
+.. attribute:: Migration.atomic
```

Comment:
I would expect a general topic about handling transactions by migrations, not a documentation of `Migration.atomic`, maybe:
```
On databases that support DDL transactions (SQLite and PostgreSQL), all
migration operations will run inside a single transaction by default. In
contrast, if a database don't support DDL transactions (e.g. MySQL, Oracle)
then all operations will run without a transaction.

You can prevent a migration from running in a transaction by setting the
``atomic`` attribute to ``False``. See :ref:`non-atomic-migrations` for
details.
```